server/db/models/cart.js

```diff
@@ -0,0 +1,16 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Cart = db.define('cart', {
+  candies: {
+    type: Sequelize.ARRAY(Sequelize.INTEGER),
+    defaultValue: []
+  }
+})
+
+Cart.prototype.remove = function(candy) {
+  this.candies = this.candies.filter(el => el !== candy)
+  return candy
```

Comment:
Do we return the removed candy whether it was removed or not?

Reply:
Good point...I was debating with this whether I should actually just return the new candies array instead? And change the test to check that it doesn't include the candy id that was passed to the method.